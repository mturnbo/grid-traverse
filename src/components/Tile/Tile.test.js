import React from 'react';
import { mount } from 'enzyme';
import TileComponent from './index';

const row = 2;
const col = 3;
const wrapper = mount(<TileComponent row={row} col={col} />);

test('Tile Component Renders', () => {
  expect(wrapper).toMatchSnapshot();
});

test('Tile Component - row prop', () => {
  expect(wrapper.prop('row')).toEqual(row);
});

test('Tile Component - col prop', () => {
  expect(wrapper.prop('col')).toEqual(col);
});
