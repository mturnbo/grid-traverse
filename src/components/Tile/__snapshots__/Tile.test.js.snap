// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile Component Renders 1`] = `
<Tile
  col={3}
  row={2}
>
  <div
    className="tile"
    data-col={3}
    data-row={2}
    onClick={[Function]}
  >
    2
    :
    3
  </div>
</Tile>
`;
